sopel/modules/isup.py

```diff
@@ -6,11 +6,13 @@
 from sopel import web
 from sopel.module import commands
 
+from requests.exceptions import SSLError
```

Comment:
Swap this line with the blank above it, keeping the imports together and also keeping PEP8 happy with two blank lines before a method definition.